### ISSUE TITLE
ES6 modules pass incorrectly marks `export` and `import` as invalid identifiers for ES6 class methods

### ISSUE THREAD

**Issue by DavidSouther** [google/closure-compiler#2071]
``` js
class Foo {
  export(id) {
     console.log(`Exporting ${id}`);
  }
}

(new Foo()).export(5);
```

Fails:

```
input0:2: ERROR - Parse error. identifier is a reserved word
  export(id) {
  ^
```

[debugger](https://closure-compiler-debugger.appspot.com/#input0%3Dclass%2520Foo%2520%257B%250A%2520%2520export%28id%29%2520%257B%250A%2520%2520%2520%2520%2520console.log%28%2560Exporting%2520%2524%257Bid%257D%2560%29%253B%250A%2520%2520%257D%250A%257D%250A%250A%28new%2520Foo%28%29%29.export%285%29%253B%26input1%26conformanceConfig%26externs%26refasterjs-template%26includeDefaultExterns%3D1%26CHECK_SYMBOLS%3D1%26CHECK_TYPES%3D1%26CLOSURE_PASS%3D1%26LANG_IN_IS_ES6%3D1%26MISSING_PROPERTIES%3D1%26PRESERVE_TYPE_ANNOTATIONS%3D1%26PRETTY_PRINT%3D1%26TRANSPILE%3D1)

[Spec 14.3](http://www.ecma-international.org/ecma-262/6.0/#sec-method-definitions) which includes `IdentifierName`, which is different than `Identifier` in that is explicitly _does_ include reserved words. 

Other reserved words do seem to be allowed:

``` js
class Foo {
  break(id) {
     console.log(`Exporting ${id}`);
  }

  break() {}
}

(new Foo()).break(5);
```

[debugger](https://closure-compiler-debugger.appspot.com/#input0%3Dclass%2520Foo%2520%257B%250A%2520%2520break%28id%29%2520%257B%250A%2520%2520%2520%2520%2520console.log%28%2560Exporting%2520%2524%257Bid%257D%2560%29%253B%250A%2520%2520%257D%250A%250A%2520%2520break%28%29%2520%257B%257D%250A%257D%250A%250A%28new%2520Foo%28%29%29.break%285%29%253B%250A%250A%252F**%2520%250Abreak%2509do%2509in%2509typeof%250Acase%2509else%2509instanceof%2509var%250Acatch%2509export%2509new%2509void%250Aclass%2509extends%2509return%2509while%250Aconst%2509finally%2509super%2509with%250Acontinue%2509for%2509switch%2509yield%250Adebugger%2509function%2509this%2509%250Adefault%2509if%2509throw%2509%250Adelete%2509import%2509try%2509%250A*%252F%26input1%26conformanceConfig%26externs%26refasterjs-template%26includeDefaultExterns%3D1%26CHECK_SYMBOLS%3D1%26CHECK_TYPES%3D1%26CLOSURE_PASS%3D1%26LANG_IN_IS_ES6%3D1%26MISSING_PROPERTIES%3D1%26PRESERVE_TYPE_ANNOTATIONS%3D1%26PRETTY_PRINT%3D1%26TRANSPILE%3D1)

I presume it's something to do with ES6 modules because `import` also fails:

``` js
class Foo {
  import(id) {
     console.log(`Exporting ${id}`);
  }
}

(new Foo()).import(5);
```

[debugger](https://closure-compiler-debugger.appspot.com/#input0%3Dclass%2520Foo%2520%257B%250A%2520%2520import%28id%29%2520%257B%250A%2520%2520%2520%2520%2520console.log%28%2560Exporting%2520%2524%257Bid%257D%2560%29%253B%250A%2520%2520%257D%250A%257D%250A%250A%28new%2520Foo%28%29%29.export%285%29%253B%250A%250A%252F**%2520%250Abreak%2509do%2509in%2509typeof%250Acase%2509%2520%2520%2520%2520%2520%2520%2520%2520else%2509instanceof%2509var%250Acatch%2509export%2509new%2509void%250Aclass%2509extends%2509return%2509while%250Aconst%2509finally%2509super%2509with%250Acontinue%2509for%2509switch%2509yield%250Adebugger%2509function%2509this%2509%250Adefault%2509if%2509throw%2509%250Adelete%2509import%2509try%2509%250A*%252F%26input1%26conformanceConfig%26externs%26refasterjs-template%26includeDefaultExterns%3D1%26CHECK_SYMBOLS%3D1%26CHECK_TYPES%3D1%26CLOSURE_PASS%3D1%26LANG_IN_IS_ES6%3D1%26MISSING_PROPERTIES%3D1%26PRESERVE_TYPE_ANNOTATIONS%3D1%26PRETTY_PRINT%3D1%26TRANSPILE%3D1)
